src/ORM/EagerLoadable.php

```diff
@@ -206,17 +206,67 @@ public function propertyPath()
     /**
      * Sets whether or not this level can be fetched using a join.
      *
+     * @param bool $possible The value to set.
+     * @return $this
+     */
+    public function setCanBeJoined($possible)
+    {
+        $this->_canBeJoined = $possible;
+
+        return $this;
+    }
+
+    /**
+     * Gets whether or not this level can be fetched using a join.
+     *
+     * @return bool
+     */
+    public function getCanBeJoined()
```

Comment:
This is is also a bit funny. I would just not have this method and keep `canBeJoined` while deprecating the setter feature in it.